ci-operator/config/redhat-appstudio/image-controller/redhat-appstudio-image-controller-main.yaml

```diff
@@ -0,0 +1,42 @@
+build_root:
+  image_stream_tag:
+    name: redhat-appstudio-e2e-test-runner
+    namespace: ci
+    tag: latest
+images:
+- dockerfile_path: Dockerfile
+  to: redhat-appstudio-image-controller-image
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: stable
+      version: "4.10"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: image-controller-e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 1h0m0s
+    version: "4.11"
```

Comment:
should we update it to `4.12`?

I know there are other redhat-appstudio components that are targeting still 4.11, but it'd probably make sense to update them in a separate PR

Reply:
Updated the version to 4.12 in the latest commit.

Reply:
please update line 14 as well for using oc 4.12 client. otherwise lgtm

Reply:
Updated line 14 to use oc 4.12 in the latest commit.